sdk/test-utils/recorder/src/recorder.ts

```diff
@@ -216,6 +217,12 @@ export class Recorder {
    * - sanitizerOptions - Generated recordings are updated by the "proxy-tool" based on the sanitizer options provided, these santizers are applied only in "record" mode.
    */
   async start(options: RecorderStartOptions): Promise<void> {
+    if (isBrowser && isPlaybackMode()) {
```

Comment:
Were changes in the recorder related to the upgrade or something found in passing?

Reply:
Yes there's some behavior changes in headers being sent (Accept-Language). I updated sanitizers in recorder and keyvault. However some keyvault tests still failed.

Reply:
I need to re-record keyvault tests with the new service version some time in the next few days (it's not fully deployed yet). Hopefully just rerecording will fix the problem but if not I will investigate further at that time :)

Reply:
Also, I am curious what the change to `Accept-Language` was. I remember this being a bugbear for me when playing back tests in the past since the locale on my machine was `en-CA` instead of `en-US` which is what was present in the recordings.

Reply:
IIRC ideally the header should be ignored by test-proxy.  Not sure what was the exact issue that prevented from happening.

Reply:
Their resistance was that it might have unintended consequences and is disruptive keeping other languages as well in mind.

From JS perspective, it should be safe, and we need to move forward with this change, so doing it in JS recorder makes sense.